packages/components/src/components/SegmentedControl/SegmentedControl.stories.tsx

```diff
@@ -45,6 +45,22 @@ export const TextOnly: Story = {
     ),
 };
 
+export const WithTooltip: Story = {
+    render: (args) => (
+        <SegmentedControl.Root {...args}>
+            <SegmentedControl.Item tooltip="First One" value="first">
+                First
+            </SegmentedControl.Item>
```

Comment:
Not much in the mindset of all the components we have
```suggestion
            <Tooltip.Root>
              <Tooltip.Trigger asChild>
                <SegmentedControl.Item tooltip="First One" value="first">
                  First
                </SegmentedControl.Item>
              </Tooltip.Trigger>
              ....
```

Let's fix it the other way around (aka in Tooltip comp) if there is an issue